x-pack/plugins/uptime/public/components/queries/monitor_status_bar/monitor_status_bar.tsx

```diff
@@ -54,21 +64,51 @@ export const MonitorStatusBar = ({
                     color={monitor.status === 'up' ? 'success' : 'danger'}
                     style={{ lineHeight: 'inherit' }}
                   >
-                    {monitor.status === 'up' ? 'Up' : 'Down'}
+                    <FormattedMessage
+                      id="xpack.uptime.monitorStatusBar.healthStatusMessage"
+                      values={{ status: monitor.status === 'up' ? 'Up' : 'Down' }}
```

Comment:
The same as in similar case above.

Reply:
Resolved in https://github.com/elastic/kibana/pull/28183/commits/93f63e384ca82a18c4419bdaa51c198a6ae5b796.